node/pkg/watchers/cosmwasm/watcher.go

```diff
@@ -158,6 +158,7 @@ func (e *Watcher) Run(ctx context.Context) error {
 
 	logger.Info("connecting to websocket", zap.String("network", networkName), zap.String("url", e.urlWS))
 
+	//nolint:bodyclose // The close is down below. The linter misses it.
```

Comment:
This seems like a bug in the linter. The `defer Close` should always come after checking for an error. 🤷